crates/meroctl/src/cli/config.rs

```diff
@@ -0,0 +1,193 @@
+use std::fs;
+use std::net::IpAddr;
+
+use calimero_network::config::BootstrapNodes;
+use clap::{Args, Parser, ValueEnum};
+use eyre::eyre;
+use multiaddr::{Multiaddr, Protocol};
+use toml_edit::{DocumentMut, Value};
+use tracing::info;
+
+use crate::cli;
+
+/// Configure the node
+#[derive(Debug, Parser)]
+pub struct ConfigCommand {
+    /// List of bootstrap nodes
+    #[arg(long, value_name = "ADDR", conflicts_with = "boot_network")]
+    pub boot_nodes: Vec<Multiaddr>,
+
+    /// Use nodes from a known network
+    #[arg(long, value_name = "NETWORK", conflicts_with = "boot_nodes")]
+    pub boot_network: Option<BootstrapNetwork>,
+
+    /// Host to listen on
+    #[arg(long, value_name = "HOST", use_value_delimiter = true)]
+    pub swarm_host: Vec<IpAddr>,
+
+    /// Port to listen on
+    #[arg(long, value_name = "PORT")]
+    pub swarm_port: Option<u16>,
+
+    /// Host to listen on for RPC
+    #[arg(long, value_name = "HOST", use_value_delimiter = true)]
+    pub server_host: Vec<IpAddr>,
+
+    /// Port to listen on for RPC
+    #[arg(long, value_name = "PORT")]
+    pub server_port: Option<u16>,
+
+    #[command(flatten)]
+    pub mdns: Option<MdnsArgs>,
+
+    /// Print the config file
+    #[arg(long, short)]
+    pub print: bool,
+}
+
+#[derive(Debug, Args)]
+#[group(multiple = false)]
+pub struct MdnsArgs {
+    /// Enable mDNS discovery
+    #[arg(long)]
+    pub mdns: bool,
+
+    #[arg(long, hide = true)]
+    pub _no_mdns: bool,
+}
+
+#[derive(Clone, Debug, ValueEnum)]
+pub enum BootstrapNetwork {
+    CalimeroDev,
+    Ipfs,
+}
+
+impl ConfigCommand {
+    pub fn run(self, root_args: cli::RootArgs) -> eyre::Result<()> {
+        let path = root_args
+            .home
+            .join(&root_args.node_name)
+            .join("config.toml");
+
+        // Load the existing TOML file
+        let toml_str =
+            fs::read_to_string(&path).map_err(|_| eyre!("Node must be initialized first."))?;
+        let mut doc = toml_str.parse::<DocumentMut>()?;
+
+        if self.print {
+            println!("{}", doc.to_string());
```

Comment:
this will call `Display::fmt` twice, you should instead delegate that call to `println`

```suggestion
            println!("{}", doc);
```